docs/ansible/roles/telegraf/real-world-example.rst

```diff
@@ -0,0 +1,124 @@
+.. Copyright (C) 2020 DebOps <https://debops.org/>
+.. SPDX-License-Identifier: GPL-3.0-or-later
+
+Real world use
+==============
+
+Telegraf requires at least one input and output defined. 
+Default ``debops.telegraf`` role variables create two plugins with nearly zero
+practical use, these are dummy input and output plugins only to fullfill minimal
+Telegraf configuration requirements. 
+
+These dummy examples are inactivated automatically if you declare any content
+to variable ``telegraf__configuration``.
+
+In next few paragraphs it will be shown real world use example.
+
+.. _telegraf__ref_options:
+
+Configuration
+-------------
+
+Telegraf instance consists of four types of plugins: ``input``, ``output``,
+``aggregator`` and ``processor`` ones. Only one input and one output is mandatory,
+but you are able to define unlimited amount of plugins in one instance. 
+
+Syntax
+~~~~~~
+
+The variables are YAML lists, each list entry is a YAML dictionary that uses
+specific parameters:
+
+``name``
+    Required. This parameter defines the option name, and it needs to be unique
+    in a given configuration file. This name will represent filename in telegraf
+    configuration directory.
+
+    It is recommended to avoid using whitespaces and other special characters, 
+    use just plain alphabet and numbers and ``_``.
+
+``config``
+    Required. It's a TOML syntax which defines a plugin according to Telegraf
+    documentation. It's just being copied without modifification into
+    configuration directory as a separate file. 
+
+    For more information, refer to the Telegraf documentation at
+    https://docs.influxdata.com/telegraf/
+
+``state``
+    Optional. By default ``present``. If you declare it as ``absent``, this
+    plugin will be removed from your Telegraf instance configuration and
+    disactivated.
+
+If you configure your instance and later remove one or more of plugins from your
+ansible configuration, it will be automatically disactivated in the host
+and backuped to a filename ending ``.inactive``, for your future reference.  
+
+Examples
```

Comment:
Moving the "Examples" section before the "Syntax" section can quickly show users that already know Telegraf how to define configuration using the DebOps role, and to easily refresh their memory in the future.